pkg/models/subscription.go

```diff
@@ -28,7 +28,7 @@ import (
  * Subscription struct
  */
 type Subscription struct {
-	BaseObject                                   `bson:",inline"`
+	BaseObject           `bson:",inline"`
```

Comment:
Looks confusing, needs indenting and alignment to other fields.

Reply:
I agree with you, looks confusing, but otherwise gofmt fails... also there are several other structs (like Addressable or Command) with the same "confusing" structure, so I think it's better to not modify again this line.

Reply:
OK, valid points.